src/components/tl/nccl/tl_nccl_context.c

```diff
@@ -93,6 +93,33 @@ static ucc_mpool_ops_t ucc_tl_nccl_req_mapped_mpool_ops = {
     .obj_cleanup   = NULL
 };
 
+static ucc_status_t ucc_tl_nccl_managed_mpool_chunk_malloc(ucc_mpool_t *mp,
+                                                           size_t *size_p,
+                                                           void ** chunk_p)
+{
+    cudaError_t cu_st;
+
+    cu_st = cudaMallocManaged((void**)chunk_p, *size_p, cudaMemAttachGlobal);
```

Comment:
as we discussed during last WG lets use explicit host mapped mem instead of managed